src/base-client.ts

```diff
@@ -11,7 +11,7 @@ export class BaseSlackAPIClient {
 
   async apiCall(
     method: string,
-    data: SlackAPIMethodArgs,
+    data: SlackAPIMethodArgs = {},
```

Comment:
adding a default here so we can treat it as optional for methods that have no required arguments